cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/StaticVarCompensatorConversion.java

```diff
@@ -46,6 +46,14 @@ public void convert() {
         context.regulatingControlMapping().forStaticVarCompensators().add(svc.getId(), p);
     }
 
+    private double getB(double rating, String name) {
+        if (rating == 0.0) {
+            fixed(name + "Rating", "Undefined or equal to 0. Corresponding susceptance is Double.MAX_VALUE");
+            return Double.MAX_VALUE;
```

Comment:
in case of bmin should be -Double.MAX_VALUE

Reply:
Yes, inductive ratings are given as negative numbers.